worlds/tracker/TrackerClient.py

```diff
@@ -188,6 +189,19 @@ class TrackerGameContext(CommonContext):
     cached_slot_data: list[dict[str, Any]] = []
     ignored_locations: set[int]
     location_alias_map: dict[int, str] = {}
+    local_items: list[NetworkItem] = []
+
+    @property
+    def tracker_items_received(self):
+        if self.items_handling == ITEMS_HANDLING:
+            return self.items_received
+        else:
+            return self.items_received + self.local_items
+
+    def update_tracker_items(self):
+        self.local_items = [self.locations_info[location] for location in self.checked_locations
+                            if location in self.locations_info and
+                            self.locations_info[location].player == self.player_id]
```

Comment:
```suggestion
                            self.locations_info[location].player == self.slot]
```
player_id is the slot number in self.multiworld not the slot number on the server